src/main/java/com/karakalpak/lingo/controller/MainController.java

```diff
@@ -0,0 +1,15 @@
+package com.karakalpak.lingo.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/")
+public class MainController {
+
+    @GetMapping("/hello")
+    public String hello() {
+        return "Hello";
```

Comment:
Why have you added this method?

Reply:
Updated this method as you suggested in [08d25e8](https://github.com/ilyastuit/lingo/pull/1/commits/08d25e8a8ed04e32c8caa9b2c4b079855666f33b)